### PR TITLE
Css variables additions

### PR DESCRIPTION
**Pull request by helendurranttw**
☝ This title will appear in the changelog - keep it meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint).

## ✅ Checklist

- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/dateLookup/DateLookup.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import (reference) '@transferwise/neptune-tokens/colors-base.less';
+@import (reference) '@transferwise/neptune-css/src/less/variables/_colors.less';
```
Comment (helendurranttw) [inline packages/components/src/dateLookup/DateLookup.less:1]: I've added a couple of semantic less variables to _colors to live until we add them to tokens. This done, I thought it best to standardise on importing all colours from there, not from tokens, for now.

File: packages/components/src/flowNavigation/backButton/BackButton.less [hunk 2]
```diff
@@ -64,17 +63,17 @@
 
   &-enter-active {
     opacity: 1;
-    transform: translateX(0px);
```
Comment (helendurranttw) [inline packages/components/src/flowNavigation/backButton/BackButton.less:67]: Stylelint changes

File: packages/components/src/navigationOption/NavigationOption.less [hunk 3]
```diff
@@ -32,11 +32,11 @@
 
   &:hover,
   &:focus {
-    color: var(--color-text-link-hover);
+    color: @color-text-link-hover;
```
Comment (helendurranttw) [inline packages/components/src/navigationOption/NavigationOption.less:35]: These are the things we should eventually add to tokens

File: packages/css/src/less/variables/_colors.less [hunk 4]
```diff
@@ -1,48 +1,6 @@
 // LESS variables for base colors
 @import '@transferwise/neptune-tokens/colors-base.less';
-// CSS custom properties
-@import '@transferwise/neptune-tokens/colors.css';
```
Comment (helendurranttw) [inline packages/css/src/less/variables/_colors.less:4]: This is a breaking change, but we shouldn't ship with custom props. If people want them they should import them themselves. We should document this..

File: packages/css/src/less/variables/_colors.less [hunk 5]
```diff
@@ -1,48 +1,6 @@
...
 
-:root {
```
Comment (helendurranttw) [inline packages/css/src/less/variables/_colors.less:6]: Ok maybe I'm being a bit hasty - we should probably add a "legacy-custom-props" file to hold all the props I deleted.

Comment (helendurranttw) [reply packages/css/src/less/variables/_colors.less:6]: Ok, added

File: packages/docs/static/assets/main.less [hunk 6]
```diff
@@ -170,24 +151,23 @@ img {
 .Flex__Container,
 .Box__Container {
   overflow: hidden;
-
   width: 100%;
 }
 
 /* Live editor */
 .live-provider {
-  margin: calc(var(--spacer) * 3) 0 0;
```
Comment (helendurranttw) [inline packages/docs/static/assets/main.less:179]: I mean... I could leave it as a calc.. but why?

File: packages/css/src/less/grid-layout/_grid-layout-ie11.less [hunk 7]
```diff
@@ -14,7 +14,7 @@
 
     > * {
       width: 100%;
-      margin-bottom: calc(var(--spacer) * 2);
+      margin-bottom: calc(var(--space-8) * 2);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/_grid-layout-ie11.less:17]:
```suggestion
      margin-bottom: var(--space-16);
```

File: packages/css/src/less/grid-layout/_grid-layout-ie11.less [hunk 8]
```diff
@@ -35,7 +35,7 @@
   screen and (min-width: @screen-md) and (-ms-high-contrast: none) {
   .d-grid {
     > * {
-      margin-bottom: calc(var(--spacer) * 4);
+      margin-bottom: calc(var(--space-8) * 4);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/_grid-layout-ie11.less:38]:
```suggestion
      margin-bottom: var(--space-32);
```

File: packages/css/src/less/grid-layout/_grid-layout-no-support.less [hunk 9]
```diff
@@ -14,7 +14,7 @@
 
     > * {
       width: 100%;
-      margin-bottom: calc(var(--spacer) * 2);
+      margin-bottom: calc(var(--space-8) * 2);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/_grid-layout-no-support.less:17]:
```suggestion
      margin-bottom: var(--space-16);
```

File: packages/css/src/less/grid-layout/_grid-layout-no-support.less [hunk 10]
```diff
@@ -33,7 +33,7 @@
   @media (-ms-high-contrast: active), screen and (min-width: @screen-md) and (-ms-high-contrast: none) {
     .d-grid {
       > * {
-        margin-bottom: calc(var(--spacer) * 4);
+        margin-bottom: calc(var(--space-8) * 4);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/_grid-layout-no-support.less:36]:
```suggestion
        margin-bottom: var(--space-32);
```

File: packages/css/src/less/grid-layout/grid-layout.less [hunk 11]
```diff
@@ -198,11 +198,11 @@
 
 [class^='grid--'],
 [class*=' grid--'] {
-  grid-gap: calc(var(--spacer) * 4);
+  grid-gap: calc(var(--space-8) * 4);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/grid-layout.less:201]:
```suggestion
  grid-gap: var(--space-32);
```

File: packages/css/src/less/grid-layout/grid-layout.less [hunk 12]
```diff
@@ -198,11 +198,11 @@
...
 
   [class^='grid--'],
   [class*=' grid--'] {
-    grid-gap: calc(var(--spacer) * 2);
+    grid-gap: calc(var(--space-8) * 2);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/grid-layout.less:205]:
```suggestion
    grid-gap: var(--space-16);
```

File: packages/css/src/less/grid-layout/grid-layout.less [hunk 13]
```diff
@@ -223,7 +223,7 @@
 }
 
 .grid-row-gap--6 {
-  grid-row-gap: calc(var(--spacer) * 6);
+  grid-row-gap: calc(var(--space-8) * 6);
```
Comment (anttio) [inline packages/css/src/less/grid-layout/grid-layout.less:226]:
```suggestion
  grid-row-gap: var(--space-48);
```

**Pull request merged**